docs/source/_topic_map.yml

```diff
@@ -14,8 +14,8 @@ Topics:
         File: installing
       - Name: Quickstart
         File: quickstart
-      - Name: Installing Docker Machine drivers
-        File: docker-machine-drivers
+      - Name: Setting up the hypervisor drivers
```

Comment:
I do not think splitting this up will be helpful, as it hides information that really belongs together. Although the driver plugins for KVM and xhyve need to be installed, they can also be configured/setup with additional options. At the moment we do not do this, but this might be added in future. The same would go for the virtualbox driver, although at this stage, no options are exposed.